NachoSpigot-Server/src/main/java/net/minecraft/server/World.java

```diff
@@ -64,7 +68,7 @@ private void guard()
     // Spigot end
     protected final Set<Entity> g = Sets.newHashSet(); // Paper
     //public final List<TileEntity> h = Lists.newArrayList(); // PaperSpigot - Remove unused list
-    public final List<TileEntity> tileEntityList = Lists.newArrayList();
+    public final me.rastrian.dev.OptimizedWorldTileEntitySet tileEntityList = new me.rastrian.dev.OptimizedWorldTileEntitySet();
```

Comment:
There is no need to qualify the entire class name, since you're importing it above.
```suggestion
    public final OptimizedWorldTileEntitySet tileEntityList = new OptimizedWorldTileEntitySet();
```